src/shared/scrapers/CAN/index.js

```diff
@@ -45,27 +46,23 @@ const scraper = {
     },
     */
     '0': async function() {
-      this.type = 'csv';
-      this.url = 'https://health-infobase.canada.ca/src/data/summary_current.csv';
       const data = await fetch.csv(this.url, false);
 
-      let scrapeDate = process.env.SCRAPE_DATE ? new Date(`${process.env.SCRAPE_DATE} 12:00:00`) : datetime.getDate();
-      let scrapeDateString = datetime.getDDMMYYYY(scrapeDate);
+      const scrapeDate = process.env.SCRAPE_DATE ? new Date(`${process.env.SCRAPE_DATE} 12:00:00`) : datetime.getDate();
+      const scrapeDateString = datetime.getDDMMYYYY(scrapeDate);
       const lastDateParts = data[data.length - 1].date.split('-');
       const lastDateInTimeseries = new Date(`${lastDateParts[2]}-${lastDateParts[1]}-${lastDateParts[0]} 12:00:00`);
       const firstDateParts = data[data.length - 1].date.split('-');
       const firstDateInTimeseries = new Date(`${firstDateParts[2]}-${firstDateParts[1]}-${firstDateParts[0]}12:00:00`);
 
       if (scrapeDate > lastDateInTimeseries) {
-        console.error(
+        throw new Error(
```

Comment:
So we're currently taking the old data anyway, not throwing. Any reason for the change?

Reply:
I'm tired of scrapers pulling old data for days with no one noticing.

I think this code will work and return results for all the dates which are in the csv. But it will throw when you ask it for a date later than what is contained in the csv.

Reply:
ℹ️  Total counts (tracked cases, may contain duplicates):
   - 10746 cases
ℹ️  Total counts (tracked cases, may contain duplicates):
   - 12484 cases
ℹ️  Total counts (tracked cases, may contain duplicates):
   - 12608 cases
  ❌ Error processing CAN:  Error:   🚨 timeseries for CAN: SCRAPE_DATE 31-03-2020 is newer than last sample time 2020-3-30.
ℹ️  Total counts (tracked cases, may contain duplicates):
   - 0 cases

Reply:
Ok, I hear that. Unfortunately it will mean that Canada will disappear from the dataset for a day if they're slow to update, which means they won't show up on the map... Maybe we should just be more aggressive about the warning? I know the log level work @ryanblock is doing will make it so we only see errors, so that will help for human eyes, and if we implement #179 and have a QA check on whether the date has matches the current date, we can throw and let the status bot know... 

@qgolsteyn @ryanblock @camjc thoughts?

Reply:
It would disappear from `data.json/csv` if it is not updated. Ideally, I think we should still include Canada with some sort of flag to indicate the data is stale

Reply:
We will have it throw when it becomes 2 days stale. For now, I am reverting to always accepting and merging this tonight, it's late and we can fix it tomorrow.